src/EditorFeatures/Core.Wpf/EditorFeaturesWpfResources.resx

```diff
@@ -153,4 +153,7 @@
   <data name="Interactive_host_process_platform" xml:space="preserve">
     <value>Interactive host process platform</value>
   </data>
+  <data name="Enter_to_rename_shift_enter_to_preview" xml:space="preserve">
+    <value>Enter to rename, Shift+Enter to preview</value>
```

Comment:
Does this need a note that Enter and Shift+Enter are keyboard keys? Not sure if it would change how it is translated.